lib/Test/Nginx/Util.pm

```diff
@@ -75,7 +75,7 @@ our $ServerAddr = '127.0.0.1';
 our $ServerName = 'localhost';
 
 our $WorkerUser = $ENV{TEST_NGINX_WORKER_USER};
-if (defined $WorkerUser && $WorkerUser !~ /^\w+(?:\s+\w+)$/) {
+if (defined $WorkerUser && $WorkerUser !~ /^\w+(\s+\w+)?$/) {
```

Comment:
good catch! But I think we still need to use `(?: )` instead of `()` to avoid the overhead of a capturing regex?